packages/widget/src/constants/graz.ts

```diff
@@ -1,5 +1,17 @@
 import { WalletType } from "graz";
 
+export const okxWalletChainIdsInitialConnect = [
+  "axelar-dojo-1",
+  "celestia",
+  "cosmoshub-4",
+  "dydx-mainnet-1",
+  "injective-1",
+  "juno-1",
+  "kava_2222-10",
+  "osmosis-1",
+  "stargaze-1",
+];
+
```

Comment:
these are the only chainIds that work for okxwallet (I eliminated the keplr chainIds one by one until i got the ones that worked)